modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -28,6 +28,9 @@ Bootstrapping with {webui} is using Salt Bundle as well, so bootstrapping with {
 Using the Salt Bundle, it is no longer needed that the client provides any Python interpreter or modules.
 
 Before bootstrapping new client you need to include [package]``venv-salt-minion`` to the bootstrap repository relevant to the client with explicitly specifying [package]``venv-salt-minion`` as an additionall package on calling [command]``mgr-create-bootstrap-repo``.
```

Comment:
Drive-by edit:
```suggestion
Before bootstrapping new client you need to include [package]``venv-salt-minion`` to the bootstrap repository relevant to the client with explicitly specifying [package]``venv-salt-minion`` as an additional package on calling [command]``mgr-create-bootstrap-repo``.
```
[I think we can break this sentence into two short sentences later.]